pkg/proxyagent/agent/manifests/charts/addon-agent/templates/namespace.yaml

```diff
@@ -0,0 +1,8 @@
+{{ if .Values.includeNamespaceCreation }}
```

Comment:
another issue i found when integrating w/ the helm agent is that we have to deal w/ namespace creation explicitly in the template. conventionally in the work api, we inline the namespace object, however in a normal helm chart, namespaces are normally implicitly created using `install --create-namespace`. for now, i had an additional value `includeNamespaceCreation` which is disable upon manual chart install but enabled when it's converted to work. @zhiweiyin318 @qiujian16